docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+    binance_bot:
+        build: ./
```

Comment:
Possibly add `tty: yes` for colourized output

Reply:
it is not possible